kong/api/crud_helpers.lua

```diff
@@ -1,7 +1,30 @@
-local utils = require "kong.tools.utils"
-local cjson = require "cjson"
-local responses = require "kong.tools.responses"
-local app_helpers = require "lapis.application"
+local utils         = require "kong.tools.utils"
+local cjson         = require "cjson"
+local responses     = require "kong.tools.responses"
+local app_helpers   = require "lapis.application"
+
+
+local decode_base64 = ngx.decode_base64
+local encode_base64 = ngx.encode_base64
+local encode_args   = ngx.encode_args
+local tonumber      = tonumber
+local ipairs        = ipairs
+local next          = next
+local type          = type
+
+
+local function post_process_row(row, cb)
+  if type(cb) == "function" then
```

Comment:
we are checking the type of `cb` both here and in `paginated_set`. is this necessary?